dev/Repeater/ItemsRepeater.cpp

```diff
@@ -118,6 +118,18 @@ winrt::Size ItemsRepeater::MeasureOverride(winrt::Size const& availableSize)
         virtualContext->Indent(Indent());
 #endif
 
+        // Checking if we have an data template
+        if (m_dataTemplateReference) {
+            // Checking if it has no content
+            if (!m_dataTemplateReference.LoadContent().as<winrt::FrameworkElement>()) {
```

Comment:
Just do this once when the ItemTemplate is set to calculate the boolean value.

Reply:
Fixed with latest commit.